config.default.php

```diff
@@ -237,3 +237,10 @@
 
 // TOTP (2FA) issuer name
 define('TOTP_ISSUER', 'Kanboard');
+
+// default filter expression
+define('DEFAULT_FILTER', 'status:open');
+
+// an expression that will be added to the predefined filters
+// define('ADDITIONAL_FILTER', 'board_date:<=today');  // use this when plugin TaskBoardDate is installed
+define('ADDITIONAL_FILTER', '');  // use this otherwise
```

Comment:
Surely there's a way to detect TaskBoardDate's presence through PHP and just use ternary conditional here _like_
```php
// an expression that will be added to the predefined filters
// checks for the presence of TaskBoardDate plugin, adds a query, otherwise nothing.
define('ADDITIONAL_FILTER', isPresentTBD ? 'board_date:<=today' : '');
```
instead of relying on the System Admin ? If the SA **wants** to change anything on this line, he **will**, and the additional comment should tell him how to interpret that TC.